### DISCUSSION TITLE
Firmware load, strapping pins status

### DISCUSSION THREAD

**Discussion by VE6SRV** [dl9rdz/rdz_ttgo_sonde#464] [Q&A] [answered]
First off, I love this software and device. I am blown away by the functionality packed into such a small package.

I bought a couple LoRa32 V2.1_1.6  devices and flashed the firmware into them a couple weeks ago. I have one sitting in my house feeding daily data to sondehub.org. The other I use to recover payloads while mobile. The unit however has no location awareness.

I just received two new T-Beam SoftRF units this morning, and am trying to flash the firmware into them. I am assuming that I can use the same firmware in both types of devices.

I am using the Windows GUI ESP8266 Flash download tool V3.9.7 which was able to flash the non-GPS units. With the T-Beam, I am seeing: 

Chip sync error 
ESP8266 Chip sync error esp_sync_blocking 

in the command line window.

A window pops up as well with:

2-sync fail
Tips:
Please refer to the chip dataset and confirm the strapping pin's status is correct.

What do I need to do to get the board in a state where it will accept the firmware load?

**Comment by dl9rdz** [comment]
Using T-Beam SoftRF: Yes, the same firmware should work – there is one catch, there are two versions of the T-Beam SoftRF, one with SX1262 and one with SX1278, and only the latter (SX1278) will work.

AFAIK it is one tool for ESP8266 and ESP32 and you need to select the chip in startup. You need to select ESP32, maybe this is the error (I find it strange that it shows an ESP8266 related error)?

Besides that, did you connect any additional peripherals to the board? This can potentially interfere with the activation of flash mode.

**Reply by VE6SRV** [reply, marked as answer]
Definitely got the 1278 version... double and triple checked to make sure when ordering. Order says that is what was shipped. I did have trouble getting the first set flashed. The software is still open from when I flashed them. (Windows machine sitting on the side of my desk.)

Nothing new added to the board, just pulled it out of the shipping box and plugged in the USB cable. Going to grab lunch and try again.

Had some lunch, shut things down and started again. I think I may have chosen ESP8266 when it is supposed to be ESP 32... pretty sure I did that last time as well. Max baud rate was 921600. Faster threw errors.

Got one unit flashed and working, time to rinse and repeat.

Once again, let me express how crazy it is that you have so much packed into such a small device. One of these will get embedded into my truck at some point.
